Antonio_Ramos_Documentation

```diff
@@ -140,12 +148,86 @@ new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Ba
+
+- found the origin of ca_wac_S000_JT00_2002.csv.gz, https://lehd.ces.census.gov/data/lodes/LODES7/ca/wac/  (ask prof. buzard if she can help clarify what the content of this file is) 
+
+06/13/2018
+- returning to the issue GoodLabs_New.shp not plotting anything when added as a layer to a blank ArcMap map, when I tested GoodLabs.shp the original file, the points do appear plotted on the map. 
+
+-changed projected coordinate system (PCS) for GoodLabs.shp in ArcGis to make it compatibale with the PCS of CA_Block_New (NAD 1983 Contiguous USA Albers) the outputted file with the updated PCS is GoodLabs_Project.shp located with the above mentioned files at "G:\MAX-Filer\Research\Labs-kbuzard-S18\Admin\CARR\California\Base Data" (do you want me to outline the steps for changing the PCS in ArcGis?) 
+
+- at this point both GoodLabs.shp and CA_Block_New.shp open and are plotted correctly in arcmap. the zip code data file hoewver is not projecting properly in arcmap. 
+
+-having a lot of trouble being able to remake the zipcode with manufacturing employment shapefile. When I add the manufa emp data in python then I have issues with the CRS in arcgis where either
+1. i change the CRS in python but then it doesnt project the data properly in arcgis
+2. i do only the join in python and try to change the CRS in arcgis, once I change the CRS again the data ceases to be projected 
+3. i try to do the join in arcgis and then attempt to change CRS, can't even do the join
+- the goal right now is to get all the data projected onto arcgis properlly, because if the data isnt projecting properly then whatever is causing that issue in arcgis might also be causing the infinite simulation loop in the Local 3Stage program
+
+- wrote program ZCTA_JOIN_Manufa_EMP.py to join the manufacuturing and employment data to the zipcode shapefile 
+
+- using ZCTA_JOIN_Manufa_EMP.py I joined DEC_00_SF3_DP3_with_ann.csv to tl_2010_06_zcta500.shp and exported the result as CA_ZCTA_Man_New.shp located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Block Level Analysis"
+
+- CA_ZCTA_Man_New projects onto ArcMap when placed by itself but when GoodLabs_Project.shp and CA_Block_New.shp are added as a layers then it becomes clear that the data from the join is messing with the scaling and size of the map. I think that when we are joining the manufacturing employment data to the zip code data we are messing up the size and scaling of the map. 
```

Comment:
This sounds like it might be the CRS instead of the PRS. Have you checked the CRS?

Reply:
I am doing that today. I think you might be right because yesterday I finally managed to create a Zip code data shapefile that included manufacturing employment and was projected at the right scale, yet the python program still didnt run correctly.